commands/game/water.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
                     } else {
                         let role = await db.fetch(`role_${guy.id}`)
                         let toKill = role.toLowerCase()
-                        if (isDay != "yes") return message.channel.send("You can use your ability only during the day!")
+                        if (gamePhase % 3 == 0) return message.channel.send("You can use your ability only during the day!")
```

Comment:
Allow voting phase too?

Reply:
yes, because it is just checking if it is night and if so it replys with that message